docs: add Layer0 deployment guide

rishi-raj-jain opened the pull request:
## Check List

- [x] Others (Update, fix, translation, etc...)
  - Languages:
  - [x] `en` English

tomap commented:
Hello, thank you for the suggestion, but a real and useful "how to" with detailed steps would be better than a marketing speech :)

rishi-raj-jain commented:
Thanks for the feedback @tomap! 

I've updated the guide to include the steps involved with a deploy button, and hope now it doesn't sound like a marketing speech as I've now highlighted the uses of the platform which is it being a deployment platform, and their own EdgeJS and prefetching, and removed the marketing portion.

SukkaW commented on source/docs/one-command-deployment.md:
````diff
@@ -314,6 +314,46 @@ After a few moments, your website will be deployed.
+
+3. Deploy
+```bash
+npm run build
````
Hexo doesn't use npm run build directly. So what does `npm run build` here for?

rishi-raj-jain replied:
Thanks for pointing that out @SukkaW, was referencing to the [demo](https://github.com/layer0-docs/layer0-hexo-example/blob/main/package.json) while I forgot to include the example. Updated to run `hexo generate` instead.
